docs/README.md

```diff
@@ -26,16 +26,21 @@ The tests currently cover
 - Advection of a smoothed sphere at an angle on a *static* grid, on a *static* grid a twice the resolution, and with *AMR* covering the sphere at the effective higher resolution
 - Advection of a sharp sphere at an angle with *AMR* writing hdf5 output and comparing against a gold standard output.
 
-To execute the tests first obtain the current gold standard output
-```bash
-# from within the main parthenon directory
-wget -qO- https://pgrete.de/dl/parthenon_regression_gold_latest.tgz | tar -xz -C tst/regression/gold_standard
-```
-and afterwards run the tests, e.g., through
+To execute the tests run, e.g.,
 ```bash
 # from within the build directory (add -V fore more detailed output)
 ctest -R regression
 ```
+The gold standard files (reference solutions) used in the regression tests should automatically be downloaded during the `make` phase.
+Alternatively, you can download them by directly make the appropriate target
+```bash
+make gold_standard
+```
```

Comment:
this is obviously a huge improvement over the `wget` command. 👍

Reply:
Wouldn't `cmake --build . --target gold_standard` be the more cmakish thing to do?

Reply:
Given that I now use `file` there's no target anymore and the file are always downloaded (if out of sync). How do people feel about this?

Reply:
This seems fine. It is a little weird that there's no way to explicitly ask the build system to "update my files" but since it happens automatically every time that seems fine.